slsim/lens.py

```diff
@@ -376,7 +376,7 @@ def los_linear_distortions(self):
         """
         kappa = self.los_class.convergence
         gamma1, gamma2 = self.los_class.shear
-        return kappa, gamma1, gamma2
+        return gamma1, gamma2, kappa
```

Comment:
I'd like to have it defined the opposite way with kappa first (see also the docstrings in this routine

Reply:
Yes, I saw your PR! and changing things occordingly.